util/gcs/gcs.go

```diff
@@ -107,6 +108,22 @@ func (g *Path) SetURL(u *url.URL) error {
 	return nil
 }
 
+func (g Path) MarshalJSON() ([]byte, error) {
+	return json.Marshal(g.String())
+}
+
+func (g *Path) UnmarshalJSON(buf []byte) error {
```

Comment:
A function comment here that indicates that this is for logrus logging of the path itself (and not, say, reading the file at `Path` and unmarshaling that) would be nice

/lint

Reply:
Done